source/includes/gcp/_k8_releases.md

```diff
@@ -331,7 +355,7 @@ curl -X POST \
    -d "request_body"
 
 
-# Request example:
+# Request body example:
```

Comment:
The same comment as below, I feel like the ``` should be before the `# Request body example`

Reply:
see comment https://github.com/cloudops/cloudmc-api-docs/pull/86/files#r377863953